evil-repeat.el

```diff
@@ -131,7 +131,7 @@
      (evil-repeat-abort)))
 
 (defsubst evil-repeat-recording-p ()
-  "Return non-nil iff a recording is in progress."
```

Comment:
Not sure this is a typo. I think it's meant to be [iff](https://en.wikipedia.org/wiki/If_and_only_if)

Reply:
I agree. OTOH, the GNU Emacs [Tips for Documentation Strings] discourage use of iff, so it should be fine either way.

[Tips for Documentation Strings]: https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html

Reply:
Heh, particularly relevant:
> Avoid using “iff” (a mathematics term meaning “if and only if”), since many people are unfamiliar with it and **mistake it for a typo**.